services/search/testing/test_search_engine.py

```diff
@@ -9,8 +10,9 @@ def test_same_weights_for_all():
     """
     result = {1: 3}
 
-    assert result == SearchEngine().search_query(constants.KEYWORDS_ARTICLE_1,
-                                                 constants.SYNONYMS_ARTICLE_1)
+    with mock.patch('requests.post', return_value=constants.KEYWORDS_DB_MOCK_1):
+        assert result == SearchEngine().search_query(constants.KEYWORDS_ARTICLE_1,
+                                                     constants.SYNONYMS_ARTICLE_1)
```

Comment:
It would be good to also use [`assert_called_once_with`](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.assert_called_once_with) to verify that `requests.post` was called with the correct arguments. Now that you're mocking it, your code could call it with random arguments and this test would technically still pass.